internal/service/searchindex/resource_search_index.go

```diff
@@ -64,7 +61,7 @@ func returnSearchIndexSchema() map[string]*schema.Schema {
 		"analyzers": {
 			Type:             schema.TypeString,
 			Optional:         true,
-			DiffSuppressFunc: validateSearchAnalyzersDiff,
+			DiffSuppressFunc: diffSuppressJSON,
```

Comment:
Can you give us more context on why are we changing this part in the PR that adds `stored_source`?

Reply:
sure, i was going to create a new diff function for stored_source but realized that all diff functions really need the same logic

Reply:
[not blocking for merging] Ah ok, by having re-looked at it seems like you removed `validateSearchIndexMappingDiff` and `validateSearchAnalyzersDiff` and renamed them `diffSuppressJSON`

Reply:
that was actually a good catch because looks like those two methods were duplicate, nice!

nit: next time just leave a comment also for these things to make the reviewer see them quicker. Thanks!